store/bsdb/object.go

```diff
@@ -128,11 +166,34 @@ func (b *BsDBImpl) ListObjectsByObjectID(ids []common.Hash, includeRemoved bool)
 	if !includeRemoved {
 		filters = append(filters, RemovedFilter(includeRemoved))
 	}
+	for _, id := range ids {
+		var object *Object
+		bucketName, err := b.GetBucketNameByObjectID(id)
+		if err != nil {
+			continue
```

Comment:
I think err here should be recorded in log

Reply:
Good point, just added error log